packages/env/lib/wordpress.js

```diff
@@ -86,11 +86,40 @@ async function configureWordPress( environment, config, spinner ) {
 		// Ignore error.
 	}
 
-	const installCommand = `wp core install --url="${ config.env[ environment ].config.WP_SITEURL }" --title="${ config.name }" --admin_user=admin --admin_password=password --admin_email=wordpress@example.com --skip-email`;
+	const isMultisite = config.env[ environment ].multisite;
+
+	const installMethod = isMultisite ? 'multisite-install' : 'install';
+	const installCommand = `wp core ${ installMethod } --url="${ config.env[ environment ].config.WP_SITEURL }" --title="${ config.name }" --admin_user=admin --admin_password=password --admin_email=wordpress@example.com --skip-email`;
 
 	// -eo pipefail exits the command as soon as anything fails in bash.
 	const setupCommands = [ 'set -eo pipefail', installCommand ];
 
+	// Bootstrap .htaccess for multisite
```

Comment:
I did this on `init-config.js` file:

```
/**
 * Default multisite .htaccess content
 */
const multisiteHtaccessContent = `# BEGIN WordPress Multisite
RewriteEngine On
RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]
RewriteBase /
RewriteRule ^index\.php$ - [L]

# add a trailing slash to /wp-admin
RewriteRule ^wp-admin$ wp-admin/ [R=301,L]

RewriteCond %{REQUEST_FILENAME} -f [OR]
RewriteCond %{REQUEST_FILENAME} -d
RewriteRule ^ - [L]
RewriteRule ^(wp-(content|admin|includes).*) $1 [L]
RewriteRule ^(.*\.php)$ $1 [L]
RewriteRule . index.php [L]
# END WordPress Multisite`;

/**
 * Writes the .htaccess file for multisite
 *
 * @param {string} wpPath Path to WordPress installation
 * @return {Promise<void>}
 */
async function writeMultisiteHtaccess( wpPath ) {
	try {
		const htaccessPath = path.join( wpPath, '.htaccess' );
		await writeFile( htaccessPath, multisiteHtaccessContent );
		// Set 644 permissions
		await chmod( htaccessPath, '0644' );
	} catch ( error ) {
		throw new Error(
			`Failed to write multisite .htaccess: ${ error.message }`
		);
	}
}
```
On some lines later, after writeChanges check in line 114

```

		// Add multisite .htaccess if multisite is enabled
		if ( config.env.multisite ) {
			await writeMultisiteHtaccess( '/var/www/html' );
			if ( config.debug ) {
				spinner.info( 'Wrote multisite .htaccess file' );
			}
		}
```

Should we check the file permission on this part?

Reply:
Oh, funny coincidence that we should be working on the same thing on the same day! 😀 

About the file permissions: did you run into any trouble in the current setup, or are you concerned with anything? Since in this PR we are creating the htaccess file by invoking shell commands _inside_ the container, I expect the resulting file to have "reasonable" permissions and ownership — unless someone really messed up the [container's mask](https://en.m.wikipedia.org/wiki/Umask). 

In contrast, in my experience, those problems arise in scenarios where files are copied from the host to the container or when mounts are set up.

Reply:
It could be considered defensive programming, I have had file permissions issues since I started setting servers 😆 . It's true that everything by default should work and that the container's mask could be the only issue.

Reply:
I pushed 0e2b0313cdd4bd49648dd44fc564b23ed2933adf, but I'm of two minds between:
* Yes, it's defensive programming, so it's better to do it — what's the harm?
* But if this is ever needed then there's something really wrong with the container (and a fresh container at that!), so isn't this a form of [cargo cult programming](https://en.wikipedia.org/wiki/Cargo_cult_programming)?

Reply:
I don't think so. The documentation's headline reads: _wp-env lets you easily set up a local WordPress environment for building and testing plugins and themes. It’s simple to install and requires no configuration._

I'll revert. :)

Reply:
It is not 😓